plausible/docker-compose.yml

```
@@ -0,0 +1,44 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: plausible_1
```

The `plausible` service will actually be named `plausible_plausible_1` by Docker, `plausible_db` will be named `plausible_plausible_db_1`, etc. 

Right now the app won't start correctly, because the app_proxy is not pointed at the container

```suggestion
      APP_HOST: plausible_plausible_1
```